src/opPool/modules/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./attestation";
```

Comment:
Can you export all of the opPool `OperationsModule`s here and use this as the import file in `src/opPool//opPool.ts`?

Reply:
oh lol, I meant to do this, but it seems I forgot to o rest of it haha